src/pip/_internal/download.py

```diff
@@ -889,6 +890,19 @@ def unpack_url(
         write_delete_marker_file(location)
 
 
+def parse_content_disposition(content_disposition, default_filename):
+    # type: (str, str) -> str
+    """
+    Parse filename from content_disposition.
+    return default_filename if parsed result is empty
```

Comment:
"content_disposition, and return default_filename if the parsed ..." (and end with a period).